docs/2022-11-02-perf-test-automation.md

```diff
@@ -0,0 +1,73 @@
+# Performance Test Automation in the Repository
+
+Two different github actions are in place for performance related work
+
+1. Time-limited profiling runs using uploaded tar-balls
+2. Timed KEVM proof suite runs, comparing to master (for pull requests only)
+
+Both actions are triggered using comments that contain a particular string.
+Only members of the RV organization are allowed to trigger the actions, and the trigger only works for issues or pull requests (PRs) with label `(performance)`.
+
+A message with a link to the action is posted as a comment.  
+Upon completion, a second message indicating success or failure is posted as a comment.
+
+Actions are implemented as nix derivations, i.e. they run in their own isolated environment, possibly using particular versions of their dependencies.
+
+## 1. Profiling runs
+
+[github action](../.github/workflows/profiling.yaml), 
+[run-profiling.nix](../nix/run-profiling.nix)
+
+### Trigger Event
+- a comment containing the string `".tar.gz)"` (anywhere)
```

Comment:
Is this supposed to contain the `)`?

Reply:
Yes, that's on purpose. We want this to be a _link_, not just a mention of some.tar.gz like here
(if I put a closing parenthesis behind that, the action will start and then fail to extract a valid download URL)